src/server/scripts/ExilesReach/zone_exiles_reach.cpp

```diff
@@ -5330,6 +5330,562 @@ class quest_choppy_booster_mk5 : public quest_scout_chopper
     }
 };
 
+// Quest 56034 - Re-sizing the Situation "Alliance"
+// Quest 59941 - Re-sizing the Situation "Horde"
+enum ResizingQuest
+{
+    ACTOR_LINDIE_RESIZING_QUEST                     = 71366,
+    ACTOR_CORK_RESIZING_QUEST                       = 76343,
+
+    EVENT_RESIZING_FOLLOW_PLAYER                    = 1,
+    EVENT_RESIZING_RUN_HOME                         = 2,
+
+    POINT_HOME_POSITION                             = 0,
+
+    SPELL_SUMMON_LINDIE_SPRINGSTOCK_GUARDIAN_Q56034 = 305750,
+    SPELL_SUMMON_CORK_FIZZLEPOP_GUARDIAN_Q59941     = 326634,
+    SPELL_LINDIE_DESUMMON_Q56034                    = 305756,
+    SPELL_PING_LINDIE_Q56034_Q59941                 = 305754,
+};
+
+Position ResizingGuardianPosition = { 100.56077f, -2418.0713f, 90.34765f };
+
+// Entry 156749 - Lindie Springstock
+struct npc_lindie_springstock_q56034 : public ScriptedAI
+{
+    npc_lindie_springstock_q56034(Creature* creature) : ScriptedAI(creature) { }
+
+    void JustAppeared() override
+    {
+        Player* player = me->GetOwner()->ToPlayer();
+        if (!player)
+            return;
+
+        Conversation* conversation = Conversation::CreateConversation(12086, player, *player, player->GetGUID(), nullptr, false);
+        if (!conversation)
+            return;
+
+        conversation->AddActor(0, 0, player->GetGUID());
+        conversation->AddActor(ACTOR_LINDIE_RESIZING_QUEST, 1, me->GetGUID());
+        conversation->AddActor(ACTOR_CORK_RESIZING_QUEST, 2, ObjectGuid::Empty);
+        conversation->Start();
+
+        _events.ScheduleEvent(EVENT_RESIZING_FOLLOW_PLAYER, 2s);
+    }
+
+    void MovementInform(uint32 uiType, uint32 uiId) override
+    {
+        if (uiType != POINT_MOTION_TYPE)
+            return;
+
+        if (uiId != 0)
+            return;
+
+        if (Player* player = me->GetOwner()->ToPlayer())
```

Comment:
crash if owner is null (e.g. via `.npc add temp`), also casting to player is not required here